src/utils/rootkeypackage_utils/src/rootkeypackage_do_download.cpp

```diff
@@ -13,6 +13,8 @@
 
 EXTERN_C_BEGIN
 
+#define MAX_TIME_TO_DOWNLOAD_IN_SECS 1 /* hour*/ * 60 /* min*/ * 60 /* sec*/
```

Comment:
re-read the email I sent on std::chrono :-)  If you have access to it, you shouldn't need to do any math.